_data/api/pro-sante-connect.md

```diff
@@ -0,0 +1,70 @@
+---
+title: API Pro Santé Connect
+tagline: Permettre l'authentification des professionnels de la santé
```

Comment:
I prefer to use a verb that describe the most frequent use of with this API. Eg. Identifier, Authentifier or Accéder aux données

We can add a mention of RPPS or the id used to identified a "professionnel"